tools/deb/BUILD

```diff
@@ -0,0 +1,67 @@
+load("@bazel_tools//tools/build_defs/pkg:pkg.bzl", "pkg_tar", "pkg_deb")
+
+pkg_tar(
+    name = "agent-bin",
+    srcs = [
+        "//cmd/pilot-agent:pilot-agent",
+    ],
+    mode = "0755",
+    package_dir = "/usr/local/bin",
+)
+
+pkg_tar(
+    name = "discovery-bin",
+    srcs = [
+        "//cmd/pilot-discovery:pilot-discovery",
+    ],
+    mode = "0755",
+    package_dir = "/usr/local/bin",
+)
+
+pkg_tar(
+    name = "debian-data-agent",
+    extension = "tar.gz",
+    deps = [
+        ":agent-bin",
+    ],
+)
+
+pkg_tar(
+    name = "discovery-systemd",
+    srcs = ["istio-discovery.service"],
+    mode = "644",
+    package_dir = "/lib/systemd/system",
+)
+
+pkg_tar(
+    name = "debian-data-discovery",
+    extension = "tar.gz",
+    deps = [
+        ":discovery-bin",
+        ":discovery-systemd",
+    ],
+)
+
+pkg_deb(
+    name = "istio-agent",
+    architecture = "amd64",
+    built_using = "bazel",
+    data = ":debian-data-agent",
+    description_file = "description-agent",
+    homepage = "http://istio.io",
+    maintainer = "The Istio Authors <istio-dev@googlegroups.com>",
+    package = "istio-agent",
+    version = "0.2.1",
```

Comment:
should the version be parametrized / a variable here ?
(though I suppose we can also add that later when we productize/align the build with rest of istio)

also can you call this file BUILD.bazel it's clearer

(copy pasting my earlier comment which disappeared because of commit race)

Reply:
You could use `version_file` and the buildstamped file.

Reply:
k8s seems to manually update the exported version as well: 

https://github.com/kubernetes/repo-infra/blob/master/defs/deb.bzl

As with Dockerfile, we may end up using the native platform builds, we can improve after we get more
info (and mileage)